### PR TITLE
Fixed few code lines in this API example

### PR DESCRIPTION
**Pull request by RenuPatelGoogle**
Updated `axis` to `axes` as per  tf.nn.moments() actual syntax and added full alias name (tfp.mcmc.) to effective_sample_size() to execute the example successfully. Please find the replicated and fixed code [gist](https://colab.sandbox.google.com/gist/RenuPatelGoogle/9a528802d6e52e46ad9713c6391bbc5c/tfp-mcmc-effective_sample_size.ipynb#scrollTo=pW_kQ2Px3VLs) for your reference.

### CODE REVIEW

**Review by ColCarroll — APPROVED** [review]

**Comment by ColCarroll** [comment]
Thanks @RenuPatelGoogle !
